src/librustc_typeck/check/generator_interior.rs

```diff
@@ -118,8 +119,12 @@ pub fn resolve_interior<'a, 'tcx>(
     // Sort types by insertion order
     types.sort_by_key(|t| t.1);
 
+    // Store the generator types, spans and expressions into the tables for this generator.
+    let expr_hir_ids = types.iter().cloned().map(|t| t.0).collect::<Vec<_>>();
```

Comment:
This variable name is probably out of date, right?

Reply:
Fixed!